### ISSUE TITLE
[Request/Query]RE: Future support for legacy (devices)?

### ISSUE THREAD

**Issue by johnth3doe** [XiangRongLin/NewPipe-preuinified#15]
#### Describe the feature you want
It is not a feature per se, rather than a general query about this fork and if/will it be (able to provide) continuing support for certain legacy device versions (in this case 4.4). 

#### Is your feature request related to a problem? Please describe it
Yes.

It has come to my attention that [this issue](https://github.com/TeamNewPipe/NewPipe/issues/7602) and [this pull request](https://github.com/TeamNewPipe/NewPipe/pull/7613) intend to drop support for not only legacy versions of android (in this case 4.4) but potentially all legacy versions of NewPipe (according to [this comment](https://github.com/TeamNewPipe/NewPipe/pull/7613#issuecomment-1007791998)) as well.

It should be known that I am not very techn(olog)ically literate, and do not know how this will affect the future of this fork. Furthermore, the original namesake [legacy version](https://github.com/TeamNewPipe/NewPipe-legacy) has not been updated in over a year, so this fork (I thank you so much for it), is a saving grace for those that have potato-devices like I do.

#### Additional context
1) @opusforlife2 mentioned in a [comment](https://github.com/TeamNewPipe/NewPipe/issues/7602#issuecomment-1007953066) that someone from the community 

> "will need to volunteer to keep the Newpipe legacy code updated so that it keeps working with every new version of the Extractor." 

I don't know how the changes mentioned in the section above affects the extractor nor how the legacy-compatible versions will be affected (v0.21.12-legacy on this fork at the time of this request/query), but I would appreciate the ability to continue using NP(-legacy) until such time where it is otherwise impossible. 

2) @Psycho-A mentioned in a [comment](https://github.com/TeamNewPipe/NewPipe/issues/7602#issuecomment-1003682284) that 

> "There are still enough Android 4.4 devices with quite sufficient performance to run NewPipe flawlessly" 

and that
 
> "at the moment it is the ONLY working YouTube client on this OS."

#### How will you/everyone benefit from this feature?
There are (older) devices that are still NewPipe compatible/functional, and would still be able to use it; However, despite this fact, the fellow member who made [issue #7602](https://github.com/TeamNewPipe/NewPipe/issues/7602) for discussion, decided that they didn't ["see any further benefit"](https://github.com/TeamNewPipe/NewPipe/issues/7602#issuecomment-1008138354) in said discussion, then led the charge to implement the [PR](https://github.com/TeamNewPipe/NewPipe/pull/7613) that is a primary reason that I am submitting this request/query.
While there are ample reasons mentioned/discussed, the consensus ultimately suggests that the changes will be implemented; and this negatively affects users that would otherwise still be able to enjoy the benefits of NP.
___
#### Additional Notes
I would like to put forward that there were valid points provided in the all of the respective linked/mentioned comments/issues, and I admit that I have a biased opinion in this because I have a potato-device, that cannot be updated (for technical reasons), and I am unable to get a new device (for financial reasons) and therefore this change affects me.

#### PS.
Notwithstanding the foregoing, I am unaware if this is the correct place or method to put my point forward, and I apologize for any unintended due diligence, inconvenience, misunderstanding or offence at any point. 
___
### TL;DR

I would like to know if the support changes to NP will affect this fork negatively, and hoping that it is not the case.

Thank you for your time and attention.
Also, thank you again to @XiangRongLin for your continued time, effort and support towards the community and for this fork. 

Stay safe, be well.

**Comment by XiangRongLin** [comment]
Any changes to only https://github.com/TeamNewPipe/NewPipe do NOT affect this fork. Any changes to https://github.com/TeamNewPipe/NewPipeExtractor DO. 
So as long as the extractor does not change in any major way, there will not be an impact on the legacy and normal releases.
 
---

If they do change in a way that breaks compatibility I will most likely NOT keep updating either of the 2 versions. Legacy or normal.
That I will decide when the time comes depending on how much time I have at hand.

---

PS: I only read the TLDR

**Comment by johnth3doe** [comment]
@XiangRongLin, I thank you for the swift reply as well as the clarification, it is appreciated.
____
I understand, and hope that nothing too major happens that would cause such inconvenience for anyone. 
I reiterate my gratitude in your continued efforts in maintaining the versions that you do, and for the respect that you have shown towards me and the community.

That being said, please do not hesitate to be as elaborate as required when responding to me, I appreciate any knowledge that you can provide and would like to learn as much as I can so that I can provide the most pertinent information that best helps both people to solve any issue without miscommunication(s) being a factor.

Also, once more I apologize for any unintended misunderstanding or offence.
____
Thank you for your time and attention.
Stay safe, be well.
____
PS. I will keep in mind that you only read the TL;DR should we ever communicate in the future. 
____
TL;DR
1) Much Thanks. 2) Hope all will be good. 3) My kindest regards

**Comment by opusforlife2** [comment]
@johnth3doe Have you read this? https://github.com/TeamNewPipe/NewPipe/issues/7602#issuecomment-1006839893
